doc/readme.html

```diff
@@ -53,32 +53,36 @@ <h2>System requirements</h2>
     <h3>Supported platforms</h3>
 
     <p>
-      <acronym>deal.II</acronym> is mostly developed on Linux using the
-      <a href="https://gcc.gnu.org">GCC</a> compiler. However, it is
-      not platform specific and we strive to keep the source code
-      compliant with
-      the <a href="https://en.cppreference.com/w/cpp/links">C++ 2014
-      Standard</a> (see
+      <acronym>deal.II</acronym> is a platform-independent library
+      written in
+      <a href="https://en.cppreference.com/w/">C++</a>, with configuration and build support provided by
+      <a href="https://cmake.org/">CMake</a>.
+      Until deal.II version 9.5, deal.II used the C++14
+      standard; subsequent versions require C++17 (see
       also <a href="https://github.com/cplusplus/draft/blob/master/papers/n4140.pdf?raw=true">here</a>
```

Comment:
```suggestion
      also <a href="https://open-std.org/JTC1/SC22/WG21/docs/standards">here</a>
```
This document is 9 years old (dated 2014-10-07), so we should update the link. I would suggest to refer to this website: https://open-std.org/JTC1/SC22/WG21/docs/standards